docs/evaluation/combine.rst

```diff
@@ -306,8 +306,8 @@ values)::
     >>> mplot.prepare(d, mdl)
     >>> fplot.prepare(dplot, mplot)
     >>> fplot.plot()
-    >>> plt.plot(x, ystart, label='Start')
-    >>> plt.legend(loc=2)
+    >>> out = plt.plot(x, ystart, label='Start')
+    >>> out = plt.legend(loc=2)
```

Comment:
Why are you capturing the output from these two lines? Can we not just label these as `# doctest: +SKIP`?

Hmm, this leads me to question what the SKIP directive does: does it run the line but does not check the output, or does it just not run the line (I assume the former)?

If you really want to capture the output, we could use something like

1. add a semicolon at the end of the line
2. use `_ = plt....`

Reply:
There are other places where I have the same question, but I'll not litter the comments with them.

Reply:
I'm not a big fan on `_ = plt...` because `_` looks somewhat special while it really is a normal variable. In other cases (not in this example) the user might even want to do something with that variable like:
```python
out = plt.legend()
out.title = 'some text'
```
So I try to be as "normal" as I can here.